bundles/org.openhab.binding.verisure/README.md

```diff
@@ -0,0 +1,507 @@
+| installationId     | Number    | This channel reports the installation ID.                         |
+| smartPlugStatus    | Switch    | This channel is used to turn smart plug on/off.                   |
+
+### Verisure Smoke Detector
+
+#### Configuration Options
+
+*   `deviceId` - Device Id
+    *   Sensor Id. Example 5A4C35FT (Note: Verisure ID, found in the Verisure App or on the sensor itself)
+
+#### Channels
+
+([smokeDetector]) supports the following channels:
+ 
+| Channel Type ID | Item Type             | Description                                                                 | 
+|-----------------|-----------------------|-----------------------------------------------------------------------------|
+| temperature     | Number:Temperature    | This channel reports the current temperature.                               |                                                                                                                                          
+| humidity        | Number                | This channel reports the current humidity in percentage.                    |
+| humidityEnabled | Switch                | This channel reports if the Climate is device capable of reporting humidity.|
+| timestamp       | DateTime              | This channel reports the last time this sensor was updated.                 |
+| location        | String                | This channel reports the location of the device.                            |
+| installationName| String                | This channel reports the installation name.                                 |
+| installationId  | Number                | This channel reports the installation ID.                                   |
+ 
+### Verisure Water Detector
+
+#### Configuration Options
+
+*   `deviceId` - Device Id
+    *   Sensor Id. Example 5A4C35FT (Note: Verisure ID, found in the Verisure App or My Pages or on the sensor itself)
+
+#### Channels
+
+([waterDetector]) supports the following channels:
+
+| Channel Type ID | Item Type             | Description                                                  | 
+|-----------------|-----------------------|--------------------------------------------------------------|                                                                                                                                          
+| temperature     | Number:Temperature    | This channel reports the current current temperature.        |
+| timestamp       | DateTime              | This channel reports the last time this sensor was updated.  |
+| location        | String                | This channel reports the location of the device.             |
+| installationName| String                | This channel reports the installation name.                  |
+| installationId  | Number                | This channel reports the installation ID.                    |
```

Comment:
Good point, since alarms are sent as push messages to the mobile/web and the binding polls the API default every 10 minutes, there is not so much real time behaviour to be able to act on alarms.

The binding has more focused on functionality to change status of the alarm, smartlocks, smartplugs and also polling climate status from all things supporting e.g. temperature and/or humidity.

However, I've implemented trigger channels in another  branch so in my production environment I'm able to receive alams via trigger channel, but then witha delay of up to default poll time (10 min).

Since the interest of getting this binding official has been very low, the binding has been for review since february 2019 :-), I'm a litte reluctant in adding this functionality since that would probably delay the process of getting this binding official even further.

Since I have a lot if users of this binding that are very pleased with its currwnt

Reply:
I'm totally okay with it, if this will be implemented in any future version. I was just wondering. You binding will be merged after running thrugh this review.

Reply:
OK!